pennylane/devices/__init__.py

```diff
@@ -87,6 +87,7 @@
 
     defer_measurements
     transforms.broadcast_expand
+    transforms.set_shots
```

Comment:
? Why is it under device transforms?

Reply:
I added this since I thought it would be also relevant for device page, so added it in this 'other transform' section. Happy to rm if you find this unnecessary

Reply:
Not a device preprocessing transform.

Reply:
removed already